packages/octopus-fig/src/entities/octopus/octopus-layer-mask-group.ts

```diff
@@ -217,7 +217,9 @@ export class OctopusLayerMaskGroup {
 
   get meta(): Octopus['LayerMeta'] | undefined {
     const dimensions = this.dimensions
-    return { dimensions, isArtboard: true }
+    const absoluteBoundingBox = this.sourceLayer.absoluteBoundingBox
+    const absoluteRenderBounds = this.sourceLayer.absoluteRenderBounds
```

Comment:
As far as I understand both `absoluteBoundingBox` and `absoluteRenderBounds` could be `undefined`. If it's `undefined`, it shouldn't be part of result.

Reply:
absoluteBounds removed